### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by KanyonKris** [bluenviron/mediamtx#2805] [General]
I needed to publish 2 webcam streams (RTSP) to a website (WebRTC) and MediaMTX has been working very well.
To all who've created, maintained and supported MediaMTX (especially @aler9), thank you!

**Comment by aler9** [comment]
Hello @KanyonKris, thank you very much for your appreciation, i have a long queue of incoming messages so i usually read them with some delay. The aim of this project is to put aside the complexity of routing live media streams and allow users to focus on their main task, and if it has allowed you to do so, the objective has been reached :)

[a comment by an automated account is not shown]
